### ISSUE TITLE
Mail stopped to be delivered to some users

### ISSUE THREAD

**Issue by fidencio** [kata-containers/community#264]
Today @snir911 raised an issue on Slack (please, see the thread: https://katacontainers.slack.com/archives/C879ACQ00/p1649061888296169) mentioning he's been not receiving emails for our mailing list for the past 2 weeks.

This has happened in the past, in a more generalised way around an year ago or so.

@snir911 also mentioned that another community member (@bpradit) has also not received emails for the past 2 weeks, both of them are registered with their "@redhat.com" emails.

@ttx, could you please help us here?  Could you take a look if something got bounced?

Also, /cc @sunnycai1 for awareness.

**Comment by ttx** [comment]
The subscriptions are both valid and active.

We had a similar problem in the past with Red Hat addresses and after some analysis they found a problem on the Red Hat email server side, rejecting emails from the list.

Let me see if I can find a reference to that earlier occurrence.

**Comment by ttx** [comment]
Hmm, actually that was a different issue.

Here is seems the Red Hat email server considers the IP we are sending email from is questionable. We have requested our delisting from that spamlist, this should hopefully unblock soon.

**Comment by ttx** [comment]
Alright, the IP address was cleared. Email should now flow again. @snir911 @bpradipt please confirm
You can access the list archives at http://lists.katacontainers.io/pipermail/kata-dev/ to read what you missed.

**Comment by snir911** [comment]
@ttx yes! i can see an email from @ariel-adam earlier today, thank you!

**Comment by ttx** [comment]
Alright we'll consider it fixed!
